src/ngraph/pass/manager.cpp

```diff
@@ -122,6 +131,19 @@ void ngraph::pass::Manager::run_passes(shared_ptr<Function> func)
             }
         }
         index++;
+        pass_timer.stop();
+        if (profile_enabled)
```

Comment:
this is a bit inconsistent with `m_visualize` and `m_serialize`. I don't know if we want consistency here like `m_profile` which is initialized in `PassManager::PassManager`?

Reply:
This is only used in this one method.